src/type-layout.md

```diff
@@ -440,12 +440,14 @@ struct MyVariantD(MyEnumDiscriminant);
 
 > Note: `union`s with non-`Copy` fields are unstable, see [55149].
 
-Combining the `repr(C)` and a primitive representation is only defined for enums
-with fields. The primitive representation modifies the `repr(C)` by changing the
+For enums with fields, it is also possible to combining `repr(C)` and a
+primitive representation. This modifies the [`repr(C)`] by changing the
 representation of the discriminant enum to have the representation of the chosen
 primitive representation. So, if you chose the `u8` representation, then the
```

Comment:
```suggestion
representation of the discriminant enum to have the representation of the chosen
primitive. So, if you chose the `u8` representation, then the
```
At least I think that's what you mean? "the representation of the chose primitive representation" does make sense, I think.

Reply:
This strange wording is still in the latest version I think.